content/source/docs/plugin/framework/data-sources.html.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: "extend"
+page_title: "Plugin Development - Framework: Data Sources"
+description: |-
+  How to build data sources in the next-generation provider development
+  framework.
+---
+
+# Data Sources
```

Comment:
@paddycarver just flagging that this is the second place where I made a commit with changes. If we do not like them, I can revert. But I essentially just matched the format to the changes I made on resources, as the pages are similar.